test/alternator/test_manual_requests.py

```diff
@@ -174,6 +175,20 @@ def test_tags_return_empty_body(dynamodb, test_table):
     response = requests.post(req.url, headers=req.headers, data=req.body, verify=False)
     assert not response.text
 
+# Reproduces issue #16233
+def test_create_table_without_keyschema(dynamodb, test_table_s):
+    payload = '{"TableName": "alternator_Test1", "AttributeDefinitions": [{"AttributeName": "lock_key", "AttributeType": "S"}, {"AttributeName": "sort_key", "AttributeType": "S"}], "BillingMode": "PAY_PER_REQUEST"}'
+    req = get_signed_request(dynamodb, 'CreateTable', payload)
+    response = requests.post(req.url, headers=req.headers, data=req.body, verify=False)
+    assert "ValidationException" in response.text and "Missing KeySchema member" in response.text
+
+# Reproduces issue #16233
+def test_create_table_without_attribute_definition(dynamodb, test_table_s):
+    payload = '{"TableName": "alternator_Test2", "KeySchema": [{"AttributeName": "lock_key", "KeyType": "HASH"}, {"AttributeName": "sort_key", "KeyType": "RANGE"}], "BillingMode": "PAY_PER_REQUEST"}'
+    req = get_signed_request(dynamodb, 'CreateTable', payload)
+    response = requests.post(req.url, headers=req.headers, data=req.body, verify=False)
+    assert "ValidationException" in response.text and "No Attribute Schema Defined" in response.text
```

Comment:
Also here, Alternator's and DynamoDB's message will be different, and you need to allow both.